restler/restler.py

```diff
@@ -415,23 +415,33 @@ def signal_handler(sig, frame):
                             print_to_console=True)
         sys.exit(-1)
 
-    if settings.token_refresh_cmd:
-        req_collection.candidate_values_pool.set_candidate_values(
-            {
-                'restler_refreshable_authentication_token':
-                    {
-                        'token_refresh_cmd': settings.token_refresh_cmd,
-                        'token_refresh_interval': settings.token_refresh_interval
-                    }
-            }
-        )
-    else:
-        req_collection.candidate_values_pool.set_candidate_values(
-            {
-                'restler_refreshable_authentication_token':
-                    {
-                    }
-            }
+    ## Support existing authentication schema
+    token_auth_method = settings.token_authentication_method
+    restler_refreshable_authentication_token = {
+        "token_auth_method": token_auth_method,
+        "token_refresh_interval": settings.token_refresh_interval,
+    }
+    if token_auth_method == 'cmd':
+        restler_refreshable_authentication_token.update({
+            "token_refresh_cmd": settings.token_refresh_cmd,
+        })
+    elif token_auth_method == 'module':
```

Comment:
module and location methods appear to do the same thing, is this intentional?

Reply:
No, hadn't finished implementing location. Added